README.md

```diff
@@ -39,6 +39,20 @@ flowchart
     A --> B
 ```
 
+## Querying with `queryContent`
+
+To retrieve the content from the `/content` directory, a query is created with
+`queryContent()`. Content is retrieved using query methods such as `find`,
+`findOne`. Elements of
+[nuxt content](https://content.nuxtjs.org/guide/displaying/rendering) such as
+`ContentRenderer` can be used to render the received markdowns.
+
+> :information_source:
+>
+> `find`, `findOne` methods return `promise` so you need to wait with `await`.
```

Comment:
biz ne için kullandık, nerede ihtiyaç duyduk onu yazalım istersen. tema'da alt sayfaları da içerde kullanmıştık. benzer bir case yaratalım